logback-ecs-encoder/src/test/resources/spec/stacktrace_as_array_spec.json

```diff
@@ -0,0 +1,139 @@
+{
```

Comment:
The spec is synced with https://github.com/elastic/ecs-logging/blob/master/spec/spec.json. Duplicating the spec would make maintenance harder.

Could you just disable spec validation in `EcsEncoderWithStacktraceAsArrayTest`? Technically, serializing stack traces as array produces JSON that's not compliant with the spec.

Reply:
Removed that spec and the associated code. I've "disabled" the spec validation by overriding the test inside the  EcsEncoderWithStacktraceAsArrayTest class. Let me know if there is a better way of doing it